deployment_scripts/deploy_windows.ps1

```diff
@@ -242,6 +242,7 @@ function Deploy-Windows([String] $monkey_home = (Get-Item -Path ".\").FullName,
     "Updating npm"
     Push-Location -Path (Join-Path -Path $monkey_home -ChildPath $MONKEY_ISLAND_DIR | Join-Path -ChildPath "\cc\ui")
     & npm update
+    & npm ci
```

Comment:
@VakarisZ I'm not very familiar with `npm`. Should we be using `npm install` instead since this is a development environment? Do we need do `npm update` if we do `npm ci`?

Reply:
`npm ci` is similar with `npm install` https://docs.npmjs.com/cli/v6/commands/npm-ci . And `npm install` didn't help solving the issue with the dependencies.

Reply:
I encountered the same problem on Ubuntu 16.04 
![image](https://user-images.githubusercontent.com/15820737/124120144-b05e5b00-da73-11eb-93a4-3378b12f6ad1.png)

Reply:
Okay so It appears that the package-lock.json and package.json after deployment are changing which is causing the problem above. 

The steps were:
 1. Run `deploy_linux.sh`
 2. Try to run the monkey_island and see that the frontend is not build
 3. Check git status to see that `package.json` and `package-lock.json` are changed.

It is resolved by:
Checkout the changes and run `npm ci` and `npm run dist` to build the frontend.